include/swift/SIL/SILOpenedArchetypesTracker.h

```diff
@@ -44,12 +36,6 @@ class SILOpenedArchetypesTracker : public DeleteNotificationHandler {
   SILOpenedArchetypesTracker(SILOpenedArchetypesTracker &Tracker)
       : SILOpenedArchetypesTracker(Tracker.F, Tracker) {}
```

Comment:
@swiftix @eeckstein could you take a quick look and reveal the intent of having two copy constructors - if you have any suggestions I'd be happy to listen